### DISCUSSION TITLE
How to disable auto completion and setting manual completion.

### DISCUSSION THREAD

**Discussion by nekowasabi** [pwntester/octo.nvim#268] [Q&A]
Hi.

I use Octo.nvim all the time.

I have a question about the issue number and user completion feature.

Currently the completion feature completes automatically, is there any way to disable this?

Also, I would like to configure the settings to allow manual completion.

Please let me know if there is a way to set this up.

Regards.

**Comment by pwntester** [comment]
This has been implemented in https://github.com/pwntester/octo.nvim/issues/287
